app.logorrr.LogoRRR.yml

```diff
@@ -0,0 +1,31 @@
+app-id: app.logorrr.LogoRRR
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: LogoRRR
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --device=dri
+  - --filesystem=home
```

Comment:
what is `home` for?

Reply:
LogoRRR is an app to view log files - what would be the best option here for such applications? i would think it should get access to any file - but i'm happy with your suggestions!

Reply:
what log files are in home? How are they opened?